Amazon_schema.txt

```diff
@@ -0,0 +1,67 @@
+create table Addresses(AddressId int primary key identity(1,1),
+StreetAddress VARCHAR(100),
```

Comment:
whose address is this ? We are not attaching any ID here? foreign key

Reply:
create table Users(UserId int primary key identity(1,1),
Name varchar(50),
PhoneNumber nvarchar(15),
Email varchar(254) unique,
AddressId int not null,
foreign key(AddressId) references Addresses(AddressId));

users address